examples/http/sonicwall_firewall_backups_configuration.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Name: SonicWall Firewall Backup configuration
+ * Description: This Configuration Management Script Extracts the SonicWall Firewall configuration and backs it up
+ * 
+ * Communication protocol is HTTPS
+ * 
+ * Tested on SonicWall SonicOS Version 7.1.1-7058
+ * 
+ * Creates a configuration backup
+ * 
+ **/
+
+var sonicWallPort = D.getParameter('sonicWallPort')
+
+/**
+ * This function sends an HTTPS GET request to the SonicWall Firewall device to obtain the configuration backup
+ * @returns {Promise} A promise that resolves to the configuration backup
+ */
+function getConfigBackup() {
+    var d = D.q.defer()
+    var config = {
+        url: "/api/",
```

Comment:
is this URL really correct?

Reply:
No, that’s incorrect. The correct one is "/api/sonicos/config/current"